src/c_kzg_4844.h

```diff
@@ -47,7 +47,7 @@ typedef blst_fr fr_t;         /**< Internal Fr field element type */
 typedef g1_t KZGCommitment;
 typedef g1_t KZGProof;
 typedef fr_t BLSFieldElement;
-typedef uint8_t Blob[BYTES_PER_BLOB];
+typedef struct { uint8_t data[BYTES_PER_BLOB]; } Blob;
```

Comment:
Perhaps renaming `data` to `bytes` would be more meaningful in the context of a blob. After all, everything is "data" in computers.

Reply:
I agree. I like `bytes` better too. Thanks!